### PR TITLE
feat: support user context in cookies API

### PR DESCRIPTION
**Pull request by sadym-chromium**
Get `userContext` from partition or from the linked browsing context.

### CODE REVIEW

File: src/bidiMapper/domains/storage/StorageProcessor.ts [hunk 1]
```diff
@@ -79,6 +83,9 @@ export class StorageProcessor {
     try {
       await this.#browserCdpClient.sendCommand('Storage.setCookies', {
         cookies: [cdpCookie],
+        ...(partitionKey.userContext === undefined
+          ? {}
+          : {browserContextId: partitionKey.userContext}),
```
Comment (OrKoN) [inline src/bidiMapper/domains/storage/StorageProcessor.ts:88]:
```suggestion
        {browserContextId: partitionKey.userContext}
```

File: tests/conftest.py [hunk 2]
```diff
@@ -377,3 +380,14 @@ async def iframe_id(websocket, context_id: str, html_iframe_same_origin, html):
     await goto_url(websocket, iframe_id, html("<h1>FRAME</h1>"))
 
     return iframe_id
+
+
+@pytest_asyncio.fixture
+async def user_context(websocket):
```
Comment (OrKoN) [inline tests/conftest.py:386]:
```suggestion
async def create_user_context(websocket):
```

File: tests/conftest.py [hunk 3]
```diff
@@ -377,3 +380,14 @@ async def iframe_id(websocket, context_id: str, html_iframe_same_origin, html):
...
+    """Create a new user context and return its id."""
+    result = await execute_command(websocket, {
+        "method": "browser.createUserContext",
+        "params": {}
+    })
+
+    return result['userContext']
```
Comment (OrKoN) [inline tests/conftest.py:393]: shall we add auto-remove?

File: tests/conftest.py (hunk 2 above)
Comment (sadym-chromium) [reply tests/conftest.py:386]: The fixture is supposed to be used as a parameter in the test, and will already provide the result.

File: tests/conftest.py (hunk 3 above)
Comment (sadym-chromium) [reply tests/conftest.py:393]: We don't clean up the browser state between tests, so no need in this yet.

File: tests/conftest.py (hunk 2 above)
Comment (OrKoN) [reply tests/conftest.py:386]: There is a fixture called create_user_context in WPT and I think to makes sense to match the style, also `create_context` would be a similar feature.

Comment (sadym-chromium) [reply tests/conftest.py:386]: Currently, the `create_context` returns a function, which can be called in test. Al long as this fixture follows the naming pattern used in this file, I suggest moving this out of scope of this PR.

Comment (OrKoN) [reply tests/conftest.py:386]: well, I am not sure there is a pattern in this file :) there is `context_id` fixture, should this one be called `user_context_id`?

**Review by OrKoN — APPROVED** [review]

File: tests/conftest.py (hunk 2 above)
Comment (sadym-chromium) [reply tests/conftest.py:386]: `user_context_id` SGTM. Done.

**Pull request merged**